src/pages/Dashboard/Dashboard_View.tsx

```diff
@@ -117,7 +117,9 @@ const Table: React.FC<TableProps> = (props: TableProps) => {
   return (
     <div className="grid grid-cols-2">
       <div className="col-span-2">
-        <h3 className="text-lg mb-3 font-medium text-gray-900">Todays Meets</h3>
+        <h3 className="text-lg mb-3 font-medium text-gray-900">
+          Today's Meets
```

Comment:
please use `&apos;` instead of an apostrophe - seo robots and crawlers can escape the page via apostrophes so its best practice to use `&apos;`